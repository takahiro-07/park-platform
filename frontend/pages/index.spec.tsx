import { render } from '@testing-library/react'

import Home from './'

describe('Sampleコンポーネント', () => {
  test('レンダーされる', () => {
    const home = render(<Home />)
    expect(home.getByText('home')).toBeTruthy()
    expect(home).toMatchSnapshot();
  })
})
