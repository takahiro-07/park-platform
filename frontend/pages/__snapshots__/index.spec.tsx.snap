// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sampleコンポーネント レンダーされる 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="container"
      >
        <main
          class="main"
        >
          <p>
            home
          </p>
        </main>
        <footer
          class="footer"
        >
          <a
            href="https://vercel.com?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
            rel="noopener noreferrer"
            target="_blank"
          >
            Powered by
             
            <span
              class="logo"
            >
              <img
                alt="Vercel Logo"
                data-nimg="1"
                decoding="async"
                height="16"
                loading="lazy"
                src="/vercel.svg"
                srcset="/vercel.svg 1x, /vercel.svg 2x"
                style="color: transparent;"
                width="72"
              />
            </span>
          </a>
        </footer>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="container"
    >
      <main
        class="main"
      >
        <p>
          home
        </p>
      </main>
      <footer
        class="footer"
      >
        <a
          href="https://vercel.com?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
          rel="noopener noreferrer"
          target="_blank"
        >
          Powered by
           
          <span
            class="logo"
          >
            <img
              alt="Vercel Logo"
              data-nimg="1"
              decoding="async"
              height="16"
              loading="lazy"
              src="/vercel.svg"
              srcset="/vercel.svg 1x, /vercel.svg 2x"
              style="color: transparent;"
              width="72"
            />
          </span>
        </a>
      </footer>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
